test/system/250-systemd.bats

```diff
@@ -292,4 +292,62 @@ LISTEN_FDNAMES=listen_fdnames" | sort)
     run_podman network rm -f $netname
 }
 
+# NOTE - running the test locally requires the `podman-play-kube@.service` file
+# **and** the `podman` binary to be installed.
```

Comment:
Confusing comment. What do you mean by "locally" and by "installed"? Is this why the test fails when I run it?
```
Failed to start podman-play-kube@-tmp-podman_bats.vvunKk-test.yaml.service: Unit podman-play-kube@-tmp-podman_bats.vvunKk-test.yaml.service not found.
```

Reply:
Shall I change the comment?

Reply:
Can you elaborate on what you mean by "this"?

What I want to do is to test ./contrib/systemd/system/podman-play-kube@.service.  A previous comment indicated that we can't access it, so the only idea I had was to make it a requirement to have the @.service file installed.

Reply:
Sorry, I hadn't taken a full look at the PR and had not understood that `contrib/systemd/system/podman-play-kube@.service.in` is a new file in the source tree that must then be preprocessed in a way that adds absolute paths.

I don't see any way to test this. But I'll cogitate on it.

Reply:
> I don't see any way to test this.

Can you elaborate on why?  It seems like I have a knowledge gap.  At least in CI, it will work because the files are installed before running the tests.

Is there an issue in gating or somewhere else?

Reply:
System tests must run in three different contexts:
* from RPM, in which case they test `/usr/bin/podman` and a full installation
* in github CI, in which case they test `./bin/podman` (in the source tree) and cannot rely on a full installation
* on developer system, ibid

Reply:
@edsantiago, the only solution I see is to move the `.service` file under `test/system/` and create a symlink in `contrib/systemd/`.

The test would have to install/uninstall the .service file but that is something we're already doing.

Reply:
But then we're not testing what the user necessarily gets and brings us back to https://github.com/containers/podman/pull/14256#discussion_r873632710.

Reply:
That does not solve the hardcoded-path problem (`@@PODMAN@@`). It is not reasonable to have the system tests deal with that.

I'm still looking into the problem and still do not see any way to test this.

Reply:
> That does not solve the hardcoded-path problem (`@@PODMAN@@`). It is not reasonable to have the system tests deal with that.

That's something the initial version of this PR took care of. `@@PODMAN@@` can be replaced with `$PODMAN`; just need to make sure it's an absolute path.

Reply:
>That's something the initial version of this PR took care of

Only for the "running in the source tree" case, not for the "running in RPM" case. Once again, I still don't see any sane way to test this. The closest idea I have is to test for `$(dirname $(dirname $PODMAN))/contrib/.../unit.in`, then if it exists, assume we're running from source dir, do your `make`, copy it to `$UNIT_DIR`, and proceed. Scary and ugly (and I haven't tested, so there may be unexpected snags).